ballerina/Dependencies.toml

```diff
@@ -21,7 +21,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.6.3"
+version = "1.6.0"
```

Comment:
Any reason to downgrade the dependencies here?

Reply:
This was an automatic commit. Will check this out.